sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FunMatcher.kt

```diff
@@ -43,6 +43,7 @@ class FunMatcherImpl(
     // so you just need to specify the package
     val qualifier: String? = null,
     val names: Set<String> = emptySet(),
+    private val nameRegex: Regex?,
```

Comment:
It's better to have this regex accessible to keep the consistency and in case if we need this field. Also, please add the default value `null`

And I think this property deserves a comment explaining how `nameRegex` actually works. So developers won't be confused whether its "names AND nameRegex" or "names OR nameRegex"